cmake/ActsConfig.cmake.in

```diff
@@ -84,9 +84,6 @@ endif()
 
 
 # dependencies which cannot be searched in CONFIG mode
-if(PluginOnnx IN_LIST Acts_COMPONENTS)
-  find_package(OnnxRuntime REQUIRED)
-endif()
```

Comment:
why is this necessary?

Reply:
You mean why is it necessary to remove this ? If you don't then when compiling against ACTS (downstream CI) we include Onnx twice. If you ask why this was here to begin with I am not sure. @paulgessinger mention we need to remplace this by a `find_dependency` so maybe he can comments about this.

Reply:
Yes I am not sure what is happening. The error I was seeing without removing those lines was that `find_package(OnnxRuntime REQUIRED)` was called twice. But I am not sure why

Reply:
to me it looks like that `find_package` will export a target that is already exported by `autodiff` which is generically called `Core`. as far as I understand `find_dependency` would be the correct thing in `*Config.cmake`. `find_dependency` does not work because there are no config files for onnx.

I dont know what would be the correct thing to do here but since removing the `find_package` solves it I would say it is good enough for now